rclcpp_action/include/rclcpp_action/client.hpp

```diff
@@ -331,8 +331,7 @@ class Client : public ClientBase
       feedback_callback,
       result_callback,
       ignore_result,
-      promise](
-        std::shared_ptr<void> response) mutable
+      promise](std::shared_ptr<void> response) mutable
```

Comment:
This wasn't what I was talking about on the style, the weird thing is that `[this` and `goal_request` use the same indention.

Reply:
Sorry, ignore this, I didn't see that you updated the signature again in the next commit.

Reply:
Right, FWIW, I recall trying to indent these lines as you suggested but one of the linters was complaining. Maybe because I didn't drop the closing `]` to the next line (I can't remember).